core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -632,11 +624,13 @@ public void invoke() {
 
         final List<String> absoluteDockerComposeFiles = composeFiles.stream()
             .map(File::getAbsolutePath)
-            .map(MountableFile::forHostPath)
-            .map(MountableFile::getFilesystemPath)
+            .map(Objects::toString)
             .collect(toList());
-        final String composeFileEnvVariableValue = absoluteDockerComposeFiles.stream().collect(joining(UNIX_PATH_SEPERATOR + "")); // we always need the UNIX path separator
+
+        final String composeFileEnvVariableValue = absoluteDockerComposeFiles.stream()
```

Comment:
since `absoluteDockerComposeFiles` is a stream already, probably makes sense to not collect it into `List`  before joining